### DISCUSSION TITLE
Oddities in Windows version

### DISCUSSION THREAD

**Discussion by axtens** [zim-desktop-wiki/zim-desktop-wiki#1824] [Q&A]
I'm seeing some odd behaviour with the Windows version 0.74.3:

1.  When I copy text to the clipboard, the Windows+V clipboard manager does not show the text but rather "No Preview Available".
2. When I use PowerToys to segment my (very) wide screen, the Zim window doesn't respond like a normal Windows window does and won't assume the size of the segment to which it is being dragged.

Are there settings anywhere to make a Zim window work more like a standard Windows window?

**Comment by introt** [comment]
1. Does this happen when using "Copy As..."? There might not be a preview because the paste location determines the result. Full disclosure: anecdotal, haven't read the code.

2. Might be due to Zim simply not fitting in such segment. Try toggling the panels closed with `F9` and/or disabling `Show edit bar along bottom of editor` from Preferences > Interface, and then dragging the window.
